src/scene/scenelicensewriter.cpp

```diff
@@ -46,6 +46,97 @@ SceneLicenseWriter::SceneLicenseWriter()
     )
 {}
 
+void sortJson(nlohmann::json& json) {
+    std::sort(json.begin(), json.end(), []
+    (const nlohmann::json& lhs, const nlohmann::json& rhs) {
+            std::string lhsString = lhs["Name"];
+            std::string rhsString = rhs["Name"];
+            std::transform(lhsString.begin(), lhsString.end(), lhsString.begin(),
+                [](unsigned char c) { return std::tolower(c); });
+            std::transform(rhsString.begin(), rhsString.end(), rhsString.begin(),
+                [](unsigned char c) { return std::tolower(c); });
+            return rhsString > lhsString;
+        });
+}
```

Comment:
See above;  maybe time to factor this into a function that can live in the `util` folder?

Reply:
Which util folder?